chains/8453.json

```diff
@@ -675,13 +675,28 @@
         "name": "MORPHO_PUBLIC_ALLOCATOR"
     },
     {
-        "addr": "0xD791292655A1d382FcC1a6Cb9171476cf91F2caa",
+        "addr": "0x996a5b8d983FA76A1555730E01c6117D90A4E6a3",
         "name": "CYPHER_EXECUTOR",
         "isContract": false
     },
     {
-        "addr": "0xD791292655A1d382FcC1a6Cb9171476cf91F2caa",
+        "addr": "0x3cb7367aC1E6a439dA1f1717f8055f02E3C9d56e",
         "name": "CYPHER_BENEFICIARY",
         "isContract": false
+    },
+    {
+        "addr": "0xDE9255FB1A82dAAaCEFCE845dAdAf6635734FC58",
+        "name": "CYPHER_ADMIN",
+        "isContract": true
+    },
+    {
+        "addr": "0x9E3ca32B18316f022fdFae503edA7E788e112AC6",
+        "isContract": true,
+        "name": "CHYPHER_AUTO_LOAD"
```

Comment:
@anajuliabit this is misspelled and should be `CYPHER_AUTO_LOAD` - will you please correct before merging?